common/common.cpp

```diff
@@ -2089,6 +2098,12 @@ struct llama_model * llama_load_model_from_url(
         return NULL;
     }
 
+    if(params.hf_token!=NULL){
+        struct curl_slist *headers = NULL;
+        headers = curl_slist_append(headers, "Authorization: Bearer " + params.hf_token);
```

Comment:
You can do it in the line before during the initialization of the headers variable, no ?